index.html

```diff
@@ -0,0 +1,202 @@
+                            <h3 class="program-event">Workshop</h3>
+                        </div>
+                    </div>
+                    <div class="program-description">
+                        <h3 class="description-program-text">
+                            Try creating your own work using open source license rather than just watching at it.                        </h3>
+                    </div>
+                </div>
+                <div class="main-program-background">
+                    <div class="program-icon-section">
+                        <div class="main-program">
+                            <img src="./img/program_icon_05 1.svg" alt="" class="program-logo">
+                        </div>
+                        <div class="program-name">
+                            <h3 class="program-event">CC ignite</h3>
+                        </div>
+                    </div>
+                    <div class="program-description">
+                        <h3 class="description-program-text">
+                            get opportunities to network with CC affiliates around the world, also after the summit.                        </h3>
+                    </div>
+                </div>
+            </div>
+            <div class="join-us" id="join">
+                <h3 class="join-us-text">Join our Kano Tech Summit 2015</h3>
+            </div>
+            <div class="whole-program">
+                <h3 class="whole-prgram-text">See the whole program</h3>
+            </div>
+        </div>
+        <div class="key-speakers">
+            <div class="key-speakers-text">
+                <h3 class="key-speakers-text">Key Speakers</h3>
+            </div>
+            <i class="fa-solid fa-horizontal-rule line-red-main"></i>
+        </div>
+        <div id="speakers-container"></div>
+        <div class="partner" id="sponsor">
+            <div class="partner-header">
+                <h3 class="partner-text">Partners</h3>
+                <i class="fa-solid fa-horizontal-rule line-red-main"></i>
+            </div>
+            <div class="partnet-companies">
+                <ul class="partner-company-names">
+                    <li><img src="./img/mozilla 1.svg" alt="" class="company-images"></li>
+                    <li><img src="./img/googlekorea 1.svg" alt="" class="company-images"></li>
+                    <li><img src="./img/naver 1.svg" alt="" class="company-images"></li>
+                    <li><img src="./img/daumkakaologo 1.svg" alt="" class="company-images"></li>
+                    <li><img src="./img/airbnb 1.svg" alt="" class="company-images"></li>
+                </ul>
+            </div>
+        </div>
+        <footer>
+            <div class="footer">
+                <img class="global-minbar-logo" src="./img/global_minbar.png" alt="global minbar logo">
+                <h3 class="footer-text">2023 Global Minbar Creative Design. Some Rights Reserved.</h3>
+            </div>
```

Comment:
- [ ] From the project requirements, you should stick to [Cindy's design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015), But as you can see from [Cindy's design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015) in the home page on the mobile screen instead of the partner's section and the footer section we have the `more` button so please make sure that you work on that.
- [ ] While implementing the `more` button please make sure that it matches Cindy's design :pray: 

| Your Design |Cindy Design |
| :---:   | :---: | 
| ![Screenshot from 2023-03-18 20-52-57](https://user-images.githubusercontent.com/92303891/226139138-eeee0a17-9bff-41b6-9dd3-a3ee1a975fd9.png) |  ![Screenshot from 2023-03-16 09-02-43](https://user-images.githubusercontent.com/92303891/225570937-a9b80634-c429-422c-bab6-a7bb68626bd3.png) |